src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -94,13 +94,13 @@ export const actions: ActionTree<ICustomFlowState, IRootState> = {
   },
   async createVolatileEmptyChoice({state, dispatch, rootGetters}, {index}) {
     const blankResource = await dispatch('flow/flow_addBlankResourceForEnabledModesAndLangs', null, {root: true})
-    const blankExit: IBlockExitTestRequired = await dispatch('flow/block_createBlockExitWith', {
```

Comment:
This has been removed from flow-runner
```
export interface IBlockExitTestRequired extends IBlockExit {
  test: string
}
```
So I just put `IBlockExit` as we already have `test` prop below